src/controller/CHIPDeviceControllerFactory.cpp

```diff
@@ -93,6 +94,7 @@ CHIP_ERROR DeviceControllerFactory::InitSystemState()
         params.fabricTable               = mSystemState->Fabrics();
         params.operationalKeystore       = mOperationalKeystore;
         params.opCertStore               = mOpCertStore;
+        params.sessionResumptionStorage  = mSessionResumptionStorage;
```

Comment:
Not: Suggest pushing down one line after the last field.

Reply:
So just put it after certificateValidityPolicy?